src/debug_wait.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#ifndef TIMESCALEDB_DEBUG_WAIT_H_
+#define TIMESCALEDB_DEBUG_WAIT_H_
+
+#include <postgres.h>
+
+#include <storage/lock.h>
+
+/* Debug waitpoint tag for debug waitpoints.
+ *
+ * Debug waitpoints only exists in debug-built code and is intended to allow
+ * more controlled testing of the code by creating wait-points where execution
```

Comment:
```suggestion
 * more controlled testing of the code by creating waitpoints where execution
```
(previously wrote "waitpoints")